test/FountainV1.test.js

```diff
@@ -1,12 +1,10 @@
-const Fountain = artifacts.require("FountainV1");
+const Fountain = artifacts.require("Fountain");
```

Comment:
Rename to `Fountain.test.js` removing V1?

Reply:
nice